models/DataObject/ClassDefinition.php

```diff
@@ -450,9 +469,14 @@ public function save($saveDefinitionFile = true)
         $cd .= ' * @method DataObject\\'.ucfirst($this->getName())."[] load()\n";
         $cd .= ' */';
         $cd .= "\n\n";
-        $cd .= 'class Listing extends DataObject\\Listing\\Concrete {';
+        $cd .= 'class Listing extends '.$extendListingClass.' {';
         $cd .= "\n\n";
 
+        if ($this->getListingUseTraits()) {
```

Comment:
It says traits (plural), so it should allow more than one trait right? this implementation only allows one trait. I would either change the property name or split the variable on char `;` pr `,` and run it in a loop.